components/InfoPage.js

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect } from 'react';
+import { View, AsyncStorage, Text, StyleSheet, Image } from "react-native";
+import { ToggleCampus } from './ToggleCampus';
+import { Icon, Button, Right } from 'native-base';
+import { AppLoading } from 'expo';
+
+function InfoPage() {
```

Comment:
Did you use this functional component anywhere? hmmm cuz it looks like the component I was suggesting to create, but I didn't see you use it.

Reply:
No functional component needed this just was used as the layout, I removed it!